src/reward_shop.cpp

```diff
@@ -180,6 +180,19 @@ class reward_shop : public CreatureScript
                 player->SetAtLoginFlag(AT_LOGIN_CHANGE_RACE);
                 ChatHandler(player->GetSession()).PSendSysMessage("CHAT OUTPUT: Please log out for race change.");
                 break;
+            case 6: /* Level Up */
```

Comment:
instead of all these comments we could replace the numbers with enums, example:
```C++
enum ShopActions {
  ITEM = 1,
  GOLD,
  NAME_CHANGE,
  FACTION_CHANGE,
  RACE_CHANGE,
  LEVEL_UP
}
...
case LEVEL_UP:
```

removing the useless comments